src/figuresClasses.ts

```diff
@@ -1,11 +1,73 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides of a triangle cannot be less than or equal to 0');
+    }
+
+    if (a + b <= c || a + c <= b || c + b <= a) {
+      throw new Error(`Sides ${a}, ${b}, and ${c} can't form a triangle.`);
+    }
+  }
+
+  getArea(): number {
+    const square = (this.a + this.b + this.c) / 2;
+
+    return (
+      Math.floor(
+        Math.sqrt(square * (square - this.a) * (square - this.b) * (square - this.c)) * 100,
```

Comment:
The properties `a`, `b`, and `c` are not declared as class properties in the `Triangle` class. This will cause a runtime error when trying to access `this.a`, `this.b`, and `this.c` in the `getArea` method. You should declare them as class properties, similar to how `color` is declared.